content/en/docs/guides/setup/android_start/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Install phone app"
+linktitle: "Install phone app"
+date: 2023-07-24
+weight: 1
+collapsible: false
+---
+
+{{< alert title="APK download" >}}
+Download the APK following the next link: [APK](https://github.com/aldajo92/AndroidKt_JoyWebSocket/releases/download/1.0/JoyWebSocket.apk).
+{{< /alert >}}
+
+After installing the APK on your phone, connect using the Raspberry Pi 4 IP address.
```

Comment:
you forgot to mention it is possible to connect to raspberry with the hostname also.

Reply:
solved